static/bidder-info/improvedigital.yaml

```diff
@@ -7,14 +7,16 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - audio
       - native
   site:
     mediaTypes:
       - banner
       - video
+      - audio
```

Comment:
should add exemplary test for audio format

Reply:
Added exemplary test for audio format